common/queue/src/main/java/org/thingsboard/server/queue/kafka/TbKafkaSettings.java

```diff
@@ -164,9 +166,22 @@ public Properties toConsumerProps(String topic) {
         consumerPropertiesPerTopic
                 .getOrDefault(topic, Collections.emptyList())
                 .forEach(kv -> props.put(kv.getKey(), kv.getValue()));
+
+        applyBaseTopicProperties(props, topic);
+
         return props;
     }
 
+    private void applyBaseTopicProperties(Properties props, String topic) {
+        if (topic != null) {
```

Comment:
Will never be null. Also, you may inline the method

Reply:
Properties consumerProps = kafkaSettings.toConsumerProps(null);

it could be... TbKafkaConsumerStatsService

Reply:
😵